### DISCUSSION TITLE
elastic body impacting water surface

### DISCUSSION THREAD

**Discussion by tjumsl** [Anura3D/Anura3D_OpenSource#25] [General]
Can Anura3D be used to simuate elastic solid body impacting water surface problem? This seems difficult in other FSI analysis method, i am wondering whether this is possible in Anura3D

**Comment by murphy-john** [comment]
Hello Tjumsl, I'll attempt to address your question in 2 parts:
1. Anura3D is a research code developed for modeling FSI problems with the Material Point Method (MPM). Our community of developers is typically focused on Geotechnical Engineering applications. Hypothetically, yes, it is "possible" to simulate any FSI problem with a given code. However, implementing the features may be impractical and/or less efficient than other codes. Additionally, the state of understanding the physics may not be there to create a model with any numerical method suitable to provide an answer. That leads to part 2.
2. The better question might be: How well does the MPM model your problem? and then, Does Anura3D have those capabilities implemented in our MPM code? The MPM and Anura3D has been used to model impact problems but without knowing what part of the problem you want to model and the types of answers you seek, it isn't possible to say if Anura3D is suitable to your problem. It is likely you could run a model with an elastic body impacting water. It would likely take some level of effort to develop, implement, and validate your model.
